autotest/gdrivers/sentinel2.py

```diff
@@ -2413,4 +2412,62 @@ def test_sentinel2_l1c_safe_compact_3():
     assert band.DataType == gdal.GDT_Byte
 
 
+###############################################################################
+# Test opening zipped versions of S2 datasets
+
+def test_sentinel2_zipped():
+    # S2 L1C
+    if not sys.platform.startswith("win"):
```

Comment:
I don't think this if platform != win check is necessary with your new code. It was necessary for the previous code that used a "sh" invokation

Reply:
Good point. Thanks.